stdlib/universe/schema_functions_test.go

```diff
@@ -876,6 +876,60 @@ func TestDropRenameKeep_Process(t *testing.T) {
 				),
 			}},
 		},
+		{
+			name: "duplicate group key",
```

Comment:
The behavior of duplicating a group key was not defined. See the addition to SPEC.